STACpopulator/log.py

```diff
@@ -98,6 +101,12 @@ def filter(self, record: logging.LogRecord) -> bool | logging.LogRecord:
         return record.levelno <= logging.INFO
 
 
+file_handler = {
+    "class": "logging.FileHandler",
+    "level": "INFO",
```

Comment:
Would it make sense to reuse the same level as `stderr`, still defined with `DEBUG` in `logconfig`? I think if there's no `--log-level` override, they should at least "align" in what they log.